weatherAplication/SupportingFiles/en.lproj/Localizable.strings

```diff
@@ -1,4 +1,6 @@
-"button_string" = "Proceed";
+"button_string" = "Search";
 "text_field_placeholder" = "city name";
 "hour_label" = "hour";
 "weather_label" = "weather";
```

Comment:
Znowu robisz ten sam błąd- nazwa tej zmiennej to powinno być po prostu _search_ a nie _buttonString_, podobnie reszta, bo chcemy odzwierciedlić zawartość zmiennej a nie jej use case